calico_node/startup/startup.go

```diff
@@ -192,20 +200,20 @@ func determineNodeName() string {
 			panic(err)
 		}
 
-		message("******************************************************************************")
-		message("* WARNING                                                                    *")
-		message("* Auto-detecting node name.  It is recommended that an explicit fixed value  *")
-		message("* is supplied using the NODENAME environment variable.  Using a fixed value  *")
-		message("* ensures that any changes to the compute host's hostname will not affect    *")
-		message("* the Calico configuration when calico/node restarts.                        *")
-		message("******************************************************************************")
+		log.Warn("******************************************************************************")
```

Comment:
Given this doesn't need to look pretty any more, should we just contact this into a single log?

Reply:
or does it still maintain its structure in most log viewers?

Reply:
or perhaps a single log with newlines in it?  iunno

Reply:
I think it will still maintain its structure in the logs, so I erred on the side of leaving it as is.

I'll play with it.